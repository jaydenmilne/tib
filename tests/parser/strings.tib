#!args=--write-tokens
"Test string"
"test string
