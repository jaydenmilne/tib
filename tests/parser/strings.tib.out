EOL/KEYWORD with value "/n" on line 1
STRING/VALUE with value "Test string" on line 2
EOL/KEYWORD with value "/n" on line 2
STRING/VALUE with value "test string" on line 3
EOL/KEYWORD with value "/n" on line 3
EOF/KEYWORD with value "" on line 4
Total Tokens = 6
